tests/rust/mod_attr.cython.toml

```diff
@@ -0,0 +1,9 @@
+header = """
+DEF FOO = 0
+DEF BAR = 0
```

Comment:
Is there a chance we wouldn't need these, by maybe doing something smart like:

```
#ifdef __cython__
DEF FOO = 0
DEF BAR = 0
#endif 
```

I believe that should make both cython and C/C++ happy with the same toml file, right?